src/modules/accommodations/accommodations.module.ts

```diff
@@ -0,0 +1,12 @@
+import { TypeOrmModule } from '@nestjs/typeorm';
+import { Module } from '@nestjs/common';
+import { Accommodation } from './entities/accommodations.entity';
+import { AccommodationController } from './accommodations.controller';
+import { AccommodationService } from './accommodations.service';
+
+@Module({
+  imports: [TypeOrmModule.forFeature([Accommodation])],
+  controllers: [AccommodationController],
+  providers: [AccommodationService],
+})
+export class AccommodationModule {}
```

Comment:
Also, it's better if module, service, and controller names are in plurals as well, like ```AccommodationsModule, AccommodationsService, AccommodationsController```. That makes more sense, since this module/service/controller will be dealing with many accommodations, not just one. In the NestJS homework also, we did ```EventsModule```